packages/descriptors/package.json

```diff
@@ -24,7 +24,7 @@
     "dist"
   ],
   "scripts": {
-    "build-core": "tsc --noEmit && tsup-node src/index.ts --clean --sourcemap --platform neutral --target=es2020 --format esm,cjs --dts && tsup-node src/index.ts --clean --sourcemap --platform neutral --target=es2020 --format cjs --dts --minify --out-dir dist/min",
+    "build-core": "tsc --noEmit && tsup-node src/index.ts --clean --sourcemap --platform neutral --target=es2022 --format esm,cjs --dts && tsup-node src/index.ts --clean --sourcemap --platform neutral --target=es2020 --format cjs --dts --minify --out-dir dist/min",
```

Comment:
this one still has 2020

Reply:
Not really important, that package is actually never published.